.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,57 @@
+name: build
+on:
+  push:
+    branches:
+      - 'main'
+    paths-ignore:
+      - 'README.md'
+      - 'samples/**'
+      - 'charts/**'
+      - 'documentation/**'
+      - 'test/**'
```

Comment:
Let's remove path `test/**`, as the changes could result in failure.

Reply:
ok